src/components/TabSelect/_TabButton.js

```diff
@@ -15,8 +15,8 @@ const Styled = component => styled(component)`
   cursor: pointer;
   margin-bottom: -1px;
   margin-right: 5px;
-  padding: 10px 20px;
-  font-size: ${props => props.theme.fontSizes.large};
+  padding: ${props => (props.small ? '5px 10px' : '10px 20px')};
```

Comment:
Right now the content is somewhat missaligned with the tab buttons with `props.small`:

![image](https://user-images.githubusercontent.com/1216874/42023750-ea32cf16-7ac0-11e8-9111-743f956fb8a7.png)

Would be good to use the same content paddings to get something like:

![image](https://user-images.githubusercontent.com/1216874/42023809-0a9e1ba2-7ac1-11e8-950f-a9d08aa9be90.png)